roapi/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "roapi"
-version = "0.8.1"
+version = "0.9.0"
```

Comment:
do we want to bump `columnq-cli` to `0.4.0` as well?

https://github.com/roapi/roapi/blob/2ec39ddba8fff359be1d8273bf43e942df421007/columnq-cli/Cargo.toml#L1-L3

Reply:
yeah, let's do that

Reply:
sounds good, done